java/src/jmri/jmrit/display/layoutEditor/LayoutTurntable.java

```diff
@@ -1041,6 +1042,64 @@ public String getTypeName() {
         return Bundle.getMessage("TypeName_Turntable");
     }
 
+    @Override
+    @Nonnull
+    public List<String> getAnchorPoints() {
```

Comment:
This needs cleanup

getAllConnectors  getAnchorPoints are mostly the same.

getPathIdentifiers and